cpnest/cpnest.py

```diff
@@ -135,7 +135,7 @@ def __init__(self,
                                   maxmcmc,
                                   verbose     = verbose,
                                   output      = output,
-                                  poolsize    = poolsize,
+                                  poolsize    = 1,
```

Comment:
Do we want poolsize 1 by default? Won't this break the ensemble jumps?

Reply:
No, poolsize =1 only for the HMC, that needs no pool